core/templates/domain/creator_dashboard/creator-dashboard-backend-api.service.ts

```diff
@@ -74,7 +74,9 @@ interface CreatorDashboardDataBackendDict {
 
 interface CreatorDashboardData {
   dashboardStats: CreatorDashboardStats;
-  lastWeekStats: CreatorDashboardStats;
+  // 'lastWeekStats' is null if 'get_last_week_dashboard_stats()' returns
+  // more than one key-value pair in the backend.
```

Comment:
I don't understand this comment.

Reply:
Updated the comment, basically it will be null for new creators. Made the comment previously by referring this line https://github.com/oppia/oppia/blob/c89776f4857746a0f7a23e09c1ceb445f6d4bdde/core/controllers/creator_dashboard_test.py#L254-L255